data/net/minecraft/client/gui/Font.mapping

```diff
@@ -164,19 +164,20 @@ CLASS net/minecraft/client/gui/Font
 		ARG 1 text
 		ARG 2 width
 	METHOD plainSubstrByWidth (Ljava/lang/String;IZ)Ljava/lang/String;
-		COMMENT tail parameter decides if returning tail (end) or head (start) of string
+		COMMENT
```

Comment:
Empty comment should be removed.

Reply:
Oh, whoops 😛 How does one completely remove a comment in enigma? Or do I have to change the file manually?

Reply:
Yeah I think you have to remove it manually (not 100% sure but I've never found an option to remove it either).